sqfc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "inputDirs": [
+        "P:/z/ace/addons/"
```

Comment:
Can this not take relative directory?

Reply:
https://github.com/dedmen/ArmaScriptCompiler/issues/8 saidly no, or atleast not yet